apps/prairielearn/src/lib/navPageTabs.ts

```diff
@@ -45,7 +47,8 @@ export function getNavPageTabs(hasEnhancedNavigation: boolean) {
             urlSuffix: '/instance_admin/lti',
             iconClasses: 'fas fa-graduation-cap',
             tabLabel: 'LTI',
-            renderCondition: ({ authz_data }) => authz_data.has_course_permission_edit,
+            renderCondition: (resLocals) =>
+              resLocals.lti_enabled && resLocals.authz_data.has_course_permission_edit,
```

Comment:
Same note as elsewhere: we have both LTI 1.1 and LTI 1.3, and while this refers specifically to LTI 1.1, that's not obvious from the name. IMO it should be.